tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -72,6 +72,12 @@ public function testCreateNews() {
     ]);
     $media_image->save();
 
+    // Assert that the content field is required.
+    $this->drupalGet('node/add/oe_sc_news');
+    $page->fillField('Title', 'Example title');
+    $page->pressButton('Save');
+    $assert_session->pageTextContains('Content field is required.');
+
     // Create a News item.
     $this->drupalGet('node/add/oe_sc_news');
```

Comment:
I'm 90% sure this line is not needed, drupal should stay on the form when it has errors